packages/node/src/models/app-instance.ts

```diff
@@ -212,6 +205,14 @@ export class AppInstance {
     return this.json.latestNonce;
   }
 
+  public get limitOrTotal() {
+    return bigNumberify(this.json.limitOrTotal._hex);
```

Comment:
Same comment, do you need to specify `._hex`?

Reply:
yes